app/controllers/answers_controller.rb

```diff
@@ -1,6 +1,6 @@
 class AnswersController < ApplicationController
   def create
-    SetAnswer.run! answer_params
+    SetAnswer.run! answer_params.to_unsafe_h
```

Comment:
Klopt, maar het is veiliger/netter om waar mogelijk `.to_h` te gebruiken, zoals hier.